profile/flatpak.sh.in

```diff
@@ -1,7 +1,13 @@
 # @sysconfdir@/profile.d/flatpak.sh - set XDG_DATA_DIRS
 
 if [ "${XDG_DATA_DIRS#*flatpak}" = "${XDG_DATA_DIRS}" ]; then
-    XDG_DATA_DIRS="${XDG_DATA_HOME:-"$HOME/.local/share"}/flatpak/exports/share:@localstatedir@/lib/flatpak/exports/share:${XDG_DATA_DIRS:-/usr/local/share:/usr/share}"
+    OLD_XDG_DATA_DIRS=${XDG_DATA_DIRS}
```

Comment:
This isn't quite what i meant though. This will look for *any* flatpak installation being configured, and then abort them all. What would be better is to handle a "partial" configuration via env.d which would then in the profile only add the missing installations.

Reply:
I think I've decided that is beyond my pain threshold for shell. Maybe somebody else can generate the necessary script? Or alternatively, write it in some other language